rbfe_tutorial/cli_tutorial.md

```diff
@@ -49,6 +49,12 @@ we do the following:
 - Pass a PDB of the protein target (TYK2) with `-p tyk2_protein.pdb`.
 - Instruct `openfe` to output files into a directory called `network_setup`
     with the `-o network_setup` option.
+- Instruct `openfe` to only run one repeat of the alchemical simulation per
+  `quickrun` call using `--n-protocol-repeats=1`.
```

Comment:
```suggestion
  `quickrun` call using `--n-protocol-repeats 1`.
```
We're not passing `=` right?

Reply:
we are! it works either way, but i think `=1` is clearer for the docs.

Reply:
It's a bit confusing to have one thing (`--n-protocol-repeats`) use `=` but not the other (`-n`). Especially given that the latter will fail if you pass it `=`. I personally think they should both show the same behaviour.